tests/scripts/thread-cert/pktverify/packet_verifier.py

```diff
@@ -165,6 +165,12 @@ def _add_initial_vars(self):
             logging.info("add extra var: %s = %s", k, v)
             self._vars[k] = v
 
+        for i, topo in self.test_info.topology.items():
+            name = self.test_info.get_node_name(i)
+            self._vars[name + '_DR_VERSION'] = 2
```

Comment:
```suggestion
            self._vars[name + '_VERSION'] = 2
```
make it shorter since it's the thread version?